### ISSUE TITLE
Verify Integrity of treatments

### ISSUE THREAD

**Issue by z5267282** [z5267282/mve#89]
ie.

```
treatments/
  /config
    /errors
    /history
    /queue
```

all exist

**Comment by z5267282** [comment]
documented in: c0f9755

**Comment by z5267282** [comment]
Can use class method in Stateful

then create Stateful and enumerate through configs

**Comment by z5267282** [comment]
If no arg given, check all configs

**Comment by z5267282** [comment]
Cannot enumerate through all configs; some might be windows and others mac

**Comment by z5267282** [comment]
This is possible, but the configs must be divided by OS: so mac / windows

**Comment by z5267282** [comment]
eg:

```
mac         # on mac MVE_CONFIGS is set here
windows

```

**Comment by z5267282** [comment]
> If no arg given, check all configs

Not doing this,

`verify_config_integrity` works on one given config name.

it will already fail if the parent config structure does not exist.

to implement this feature i would have to reuse code from `config.py`

There is also no need for the feature.
Users should check one at a time if configs exist.

**Comment by z5267282** [comment]
> > If no arg given, check all configs
> 
> Not doing this,
> 
> `verify_config_integrity` works on one given config name.
> 
> it will already fail if the parent config structure does not exist.
> 
> to implement this feature i would have to reuse code from `config.py`
> 
> There is also no need for the feature. Users should check one at a time if configs exist.

Wait no it is possible, it is as simple as doing

```sh
for config in $MVE_CONFIGS/*
```

**Comment by z5267282** [comment]
Implemented partially in 497984d

**Comment by z5267282** [comment]
done in 4972717
